attributes/packages.rb

```diff
@@ -40,7 +40,8 @@
            { name: 'impyla', version: '>=0.13.3' },
            { name: 'unicodecsv', version: '>=0.14.1' }],
     jdbc: [{ name: 'jaydebeapi', version: '>=0.2.0' }],
-    mysql: [{ name: 'mysqlclient', version: '>=1.3.6' }],
+    mysql: [{ name: 'mysqlclient', version: '1.3.6,<1.4' }, 
+           { name: 'mysql-connector-python', version: '>=8.0.21, <=8.0.21' }],
```

Comment:
I don't think this are needed anymore. See my comment below on getting rid of them.

Reply:
removed